api/config/genesis.go

```diff
@@ -15,14 +15,15 @@ type GenesisConfig struct {
 	InitialAccounts map[address.Address]GenesisAccount
 }
 
+const Account1Pub = "0x7be017a967db77fd10ac7c891b3d6d946dea7e3e14756e2f0f9e09b9663f0d9c"
+const Account1Private = "0x81c90dd832e18d1cf9758254327cb3135961af6688ac9c2a8c5d71f73acc5ce57be017a967db77fd10ac7c891b3d6d946dea7e3e14756e2f0f9e09b9663f0d9c"
```

Comment:
this shouldnt be a parameter since it is not used in the code, this was commented to keep private key

Reply:
We use it in app_test